scripts/us_census/pep/Population_Estimate_by_ASR/county_1990_2000.py

```diff
@@ -0,0 +1,118 @@
+                    encoding='ISO-8859-1')
+            # Removing the lines that have false symbols.
+            num_df = (df.drop(cols, axis=1).join(df[cols]\
+                .apply(pd.to_numeric, errors='coerce')))
+            df = num_df[num_df[cols].notnull().all(axis=1)]
+            df[1:] = df[1:].apply(pd.to_numeric)
+            df['geo_ID'] = df['geo_ID'].astype(int)
+
+            # Providing geoId to the dataframe.
+            df['geo_ID'] = [f'{x:05}' for x in df['geo_ID']]
+            # Replacing the numbers with more understandable metadata headings.
+            df = _replace_age(df)
+
+            # Columns after 11 where having origin.
+            df.drop(df[df['Race'] >= 11].index, inplace=True)
+            # Changing the column values as per metadata.
+            df['Race'] = df['Race'].astype(int).astype(str)
+            df['Race'] = df['Race'].str.replace('10',\
+                'Female_AsianOrPacificIslander')
+            df['Race'] = df['Race'].str.replace('1',\
+                'Male_WhiteAlone')
+            df['Race'] = df['Race'].str.replace('2',\
+                'Female_WhiteAlone')
+            df['Race'] = df['Race'].str.replace('3',\
+                'Male_WhiteAlone')
+            df['Race'] = df['Race'].str.replace('4',\
+                'Female_WhiteAlone')
+            df['Race'] = df['Race'].str.replace('5','Male_BlackOrAfrican'+\
+                'AmericanAlone')
+            df['Race'] = df['Race'].str.replace('6', 'Female_BlackOrAfrican'+\
+                'AmericanAlone')
+            df['Race'] = df['Race'].str.replace('7', 'Male_AmericanIndianAnd'+\
+                'AlaskaNativeAlone')
+            df['Race'] = df['Race'].str.replace('8', 'Female_AmericanIndianAnd'\
+                +'AlaskaNativeAlone')
+            df['Race'] = df['Race'].str.replace('9', \
+                'Male_AsianOrPacificIslander')
+
+            df = df.melt(id_vars=['Year','geo_ID' ,'Race'], var_name='Age' , \
+                value_name='observation')
+            df['SVs'] = 'Count_Person_' + df['Age'] + '_' + df['Race']
+            df.drop(columns=['Race', 'Age'], inplace=True)
+            # writing the dataframe to final dataframe
+            final_df = pd.concat([final_df, df])
+
+    # Creating geoId.
+    final_df['geo_ID'] = 'geoId/' + final_df['geo_ID'].astype("str")
+    final_df = final_df.groupby(['Year', 'geo_ID', 'SVs']).sum().reset_index()
+    # Adding measurement method.
+    final_df['Measurement_Method'] = np.where(final_df['SVs'].str.contains\
+        ('White'), 'dcAggregate/CensusPEPSurvey', 'CensusPEPSurvey')
+    # Making copies and using group by to get Aggregated Values.
+    df_as = pd.concat([final_df, df_as])
+    df_ar = pd.concat([df_ar, final_df])
+    df_as['SVs'] = df_as['SVs'].str.replace\
+        ('_AmericanIndianAndAlaskaNativeAlone','')
+    df_as['SVs'] = df_as['SVs'].str.replace('_AsianOrPacificIslander', '')
```

Comment:
Similar comment to the one is the 1980_1989 file. I know _AsiansOrPacificIslander is different. But you can still abstract these in a common function, no?

Reply:
Created function for these in the common functions python file. All the files that needed this change have been changed accordingly